sky/skylet/job_lib.py

```diff
@@ -263,9 +283,12 @@ def _get_jobs_by_ids(job_ids: List[int]) -> List[Dict[str, Any]]:
 
 def update_job_status(job_owner: str,
                       job_ids: List[int],
-                      need_output: bool = True) -> List[JobStatus]:
-    """Updates the "true" job statuses from our perspective in the database,
-    i.e., with the semantics documented in the comments of `class JobStatus`.
+                      silent: bool = False) -> List[JobStatus]:
+    """Updates and returns the true job statuses that match our semantics
```

Comment:
styleguide nit: need to fit on 1 line

maybe something like: Updates and returns true job statuses matching our JobStatus semantics.